signal.go

```diff
@@ -0,0 +1,26 @@
+package gb
```

Comment:
This code should not live in the gb package, it should know nothing about it. Ideally this would live in the cmd or cmd/gb package.